internal/common/test_configuration.go

```diff
@@ -168,6 +168,14 @@ var TestIPv6Networking = TestNetworking{
 	IngressVip:      "1001:db8::65",
 }
 
+var TestEquivalentIPv6Networking = TestNetworking{
```

Comment:
not for this PR but i prefer to have a `testconfig` package instead of common package.